two-tier-app-migration-to-containers-on-Azure/scripts/DotNetAppSqlDb.deploy.cmd

```diff
@@ -0,0 +1,311 @@
+                      
+@rem ---------------------------------------------------------------------------------
+@rem CheckParameterFile -- check if the package's setparamters.xml exists or not
+@rem ---------------------------------------------------------------------------------
+:CheckParameterFile
+if exist "%_DeploySetParametersFile%" (
+echo SetParameters from:
+echo "%_DeploySetParametersFile%"
+echo You can change IIS Application Name, Physical path, connectionString
+echo or other deploy parameters in the above file.
+) else (
+echo SetParamterFiles does not exist in package location.
+echo Use package embedded defaultValue to deploy.
+)
+echo -------------------------------------------------------
+goto :eof
+
+@rem ---------------------------------------------------------------------------------
+@rem Usage
+@rem ---------------------------------------------------------------------------------
+:usage
+echo =========================================================
+if not exist "%RootPath%DotNetAppSqlDb.deploy-readme.txt" (
+echo Usage:%~nx0 [/T^|/Y] [/M:ComputerName] [/U:userName] [/P:password] [/G:tempAgent] [additional msdeploy flags ...]
+echo Required flags:
+echo /T  Calls msdeploy.exe with the "-whatif" flag, which simulates deployment. 
+echo /Y  Calls msdeploy.exe without the "-whatif" flag, which deploys the package to the current machine or destination server 
+echo Optional flags:  
+echo. By Default, this script deploy to the current machine where this script is invoked which will use current user credential without tempAgent. 
+echo.   Only pass these arguments when in advance scenario.
+echo /M:  Msdeploy destination name of remote computer or proxy-URL. Default is local.
+echo /U:  Msdeploy destination user name. 
+echo /P:  Msdeploy destination password.
+echo /G:  Msdeploy destination tempAgent. True or False. Default is false.
+echo /A:  specifies the type of authentication to be used. The possible values are NTLM and Basic. If the wmsvc provider setting is specified, the default authentication type is Basic
+otherwise, the default authentication type is NTLM.
+echo /L:  Deploy to Local IISExpress User Instance.  
+
+echo.[additional msdeploy flags]: note: " is required for passing = through command line.
+echo  "-skip:objectName=setAcl" "-skip:objectName=dbFullSql"
+echo.Alternative environment variable _MsDeployAdditionalFlags is also honored.
+echo.
+echo. Please make sure MSDeploy is installed in the box https://go.microsoft.com/?linkid=9278654
+echo.
+echo In addition, you can change IIS Application Name, Physical path, 
+echo connectionString and other deploy parameters in the following file:
+echo "%_DeploySetParametersFile%"
+echo.
+echo For more information about this batch file, visit https://go.microsoft.com/fwlink/?LinkID=183544 
+) else (
+start notepad "%RootPath%DotNetAppSqlDb.deploy-readme.txt"
+)
+echo =========================================================
+goto :eof

```

Comment:
need source for DotNetAppSqlDb.zip
remove webdeploy msi from the sample and get that file from a secure location

Reply:
removed webdeploy and provided link to download from MS.

Reply:
Application source is https://docs.microsoft.com/en-us/azure/app-service/app-service-web-tutorial-dotnet-sqldatabase and is complied in Visual Studio.